pkg/kind/kind.go

```diff
@@ -234,6 +243,26 @@ func checkForExistingCluster() error {
 	return nil
 }
 
+// recreateCluster recreates a Kind cluster
+func recreateCluster() error {
+	fmt.Println("\n    Deleting cluster...")
+	deleteCluster := exec.Command("kind", "delete", "cluster", "--name", clusterName)
+	if err := deleteCluster.Run(); err != nil {
+		return fmt.Errorf("delete cluster: %w", err)
+	}
+	deleteContainerRegistry := deleteContainerRegistry()
+	if err := deleteContainerRegistry.Run(); err != nil {
+		return fmt.Errorf("delete container registry: %w", err)
+	}
+	if err := createNewCluster(); err != nil {
+		return fmt.Errorf("new cluster: %w", err)
+	}
+	if err := connectLocalRegistry(); err != nil {
+		return fmt.Errorf("local-registry: %w", err)
```

Comment:
When trying to recreate the cluster, I got an error here with a `failed to connect local registry to kind cluster` error

Reply:
Will look into it

Reply:
Ok, I've found the issue. comitted the changes